tests/xs/opacity_Fe_test.py

```diff
@@ -0,0 +1,80 @@
+"""test for opacity calculation of metal lines
```

Comment:
In the current version,  there are 5 unit tests for Fe opacity:
```
opacity_Fe_test.py
opacity_Fe_test_gamma_KA3.py
opacity_Fe_test_gamma_KA3s.py
opacity_Fe_test_gamma_KA4.py
opacity_Fe_test_gamma_uns.py
```
It looks a lot. Can you combine these tests to the one unit test?
Instead, I recommend to reduce the number of items for each test to save the time.